packages/nimble-components/src/rich-text/editor/tests/rich-text-editor.spec.ts

```diff
@@ -343,6 +366,20 @@ describe('RichTextEditor', () => {
             expect(pageObject.getEditorLeafContents()).toEqual(['italics']);
         });
 
+        it('should have br tag name when pressing shift + Enter with Italics content', async () => {
+            await pageObject.clickFooterButton(ToolbarButton.italics);
+            await pageObject.setEditorTextContent('italics1');
+            await pageObject.pressShiftEnterKeysInEditor();
+            await pageObject.setEditorTextContent('italics2');
```

Comment:
```suggestion
            await pageObject.setEditorTextContent('italics after hard break');
```

Reply:
nit: Apply similarly for other test cases for a slightly enhanced readability

Reply:
Updated as per your suggestion.